packages/create-emotion-styled/package.json

```diff
@@ -26,10 +22,7 @@
   "devDependencies": {
     "@types/react": "16.0.16",
     "dtslint": "^0.3.0",
-    "npm-run-all": "^4.0.2",
-    "prop-types": "^15.6.1",
-    "rimraf": "^2.6.1",
-    "rollup": "^0.60.1"
+    "prop-types": "^15.6.1"
```

Comment:
seems weird to have prop-types in devDeps, are they needed here?

Reply:
It was already there. It's probably not strictly necessary but it was probably added there because it's a peerDependency and it's needed in dev. If you think it should be removed them I'll remove it but IMO, it doesn't really matter.

Reply:
peer dep thing makes sense 👍